src/sst/core/configGraphOutput.h

```diff
@@ -48,6 +49,53 @@ class ConfigGraphOutput
 
 protected:
     FILE* outputFile;
+
+    /**
+     * Get a named global parameter set.
+     *
+     * @param name Name of the set to get
+     *
+     * @return returns a copy of the reqeusted global param set
+     *
+     */
+    static std::map<std::string, std::string> getGlobalParamSet(const std::string& name)
+    {
+        return Params::getGlobalParamSet(name);
+    }
+
+
+    /**
+     * Get a vector of the names of available global parameter sets.
+     *
+     * @return returns a vector of the names of available global param
+     * sets
+     *
+     */
+    static std::vector<std::string> getGlobalParamSetNames() { return Params::getGlobalParamSetNames(); }
```

Comment:
''

Reply:
The underlying functions the function calls creates the data structures on the fly, so there isn't a live variable to return a reference to.